addon/helpers/pikaday.js

```diff
@@ -31,6 +31,12 @@ var PikadayInteractor = {
   },
   selectedYear: function() {
     return $(this.selectorForYearSelect + ' option:selected').val();
+  },
+  minYear: function() {
```

Comment:
I think abbreviations are an anti-pattern in source code so could you please change `minYear` and `maxYear` to `minimumYear` and `maximumYear`, thanks!